{'runtime': datetime.datetime(2022, 3, 13, 15, 58, 31, 945374), 'exceptionDetails': [{'year': 2019, 'month': 1, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x0000022A8655C700>)}, {'year': 2019, 'month': 2, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x0000022A8656C440>)}, {'year': 2019, 'month': 3, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x0000022A86580E40>)}]}
{'runtime': datetime.datetime(2022, 3, 13, 15, 59, 27, 365849), 'exceptionDetails': [{'year': 2019, 'month': 1, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x000001F50B7BC5C0>)}, {'year': 2019, 'month': 2, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x000001F50B7CED80>)}, {'year': 2019, 'month': 3, 'failureReason': (<class 'ValueError'>, ValueError('Error 401: Invalid API Key'), <traceback object at 0x000001F50B7E0B80>)}]}
